db/migrate/20230223081955_create_motorcycles.rb

```diff
@@ -0,0 +1,16 @@
+class CreateMotorcycles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :motorcycles do |t|
+      t.string :name
+      t.text :description
+      t.string :img_url
+      t.string :model_year
+      t.decimal :price
+      t.string :engine
+      t.string :fuel_type
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
```

Comment:
user_id is a foreign_key it needs to be indexed, do this.
`t.references :user, null: false, foreign_key: true, index: true`